01.fizzbuzz/fizzbuzz.rb

```diff
@@ -0,0 +1,13 @@
+i = 0
+20.times.each do
```

Comment:
`each`なしで
```suggestion
20.times do
```
でいけると思います！

Reply:
さらにいうと、ブロック変数を使えば`i`を上で定義する必要がなくなります。
```suggestion
20.times do |i|
```
ブロック変数`i`は0から始まって繰り返しごとに１ずつ増えていくので、下の`i += 1`も必要なくなりますね。